RecoLocalCalo/HGCalRecAlgos/src/RecHitTools.cc

```diff
@@ -114,6 +125,20 @@ GlobalPoint RecHitTools::getPosition(const DetId& id) const {
   return position;
 }
 
+GlobalPoint RecHitTools::getPositionLayer(unsigned layer) const {
+  DetId id(0);
+  if(geometryType_==0){
+    if (layer <= fhOffset_) id = HGCalDetId(ForwardSubdetector::HGCEE, 1, layer, 1, 50, 1);
+    else if (layer <= bhOffset_) id = HGCalDetId(ForwardSubdetector::HGCHEF, 1, layer - fhOffset_, 1, 50, 1);
+    else  id = HcalDetId(HcalSubdetector::HcalEndcap, 50, 100, layer - bhOffset_);
```

Comment:
is this magic 50 cast in stone?
It makes sense to make a named constant

Reply:
This was there before, but I'll make it a named const now